src/sql/base/browser/ui/table/plugins/media/checkboxSelectColumn.plugin.css

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.slick-plugin-checkbox-select-column {
+	text-align: center;
+	border-bottom: 0px !important;
```

Comment:
can you use more specific css selectors to avoid needing to use `important` modifier?